pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/route/GetCancerSuggestionsRoute.java

```diff
@@ -33,53 +35,53 @@ public class GetCancerSuggestionsRoute implements Route {
     private static final String CANCER_QUERY_REGEX = "\\w+";
     private static final int DEFAULT_LIMIT = 100;
     private final CancerStore cancerStore;
-    
 
     @Override
     public CancerSuggestionResponse handle(Request request, Response response) {
+        String cancerListLanguage = RouteUtil.getUserLanguage(request).getIsoCode();
```

Comment:
If no language in the request, we are using study default  (which is always en ) rather than querying from DB table user_profile which has users preferred language ?